web-registry/src/lib/bridge.ts

```diff
@@ -17,7 +17,31 @@ export const getBridgeTxStatus = async (
     return;
   }
   try {
-    const { data } = await axios.get(`${apiUri}/regen/events/${hash}/status`);
+    const { data } = await axios
+      .get(`${apiUri}/regen/events/${hash}/status`)
+      .catch(error => {
+        if (error.response) {
+          // The request was made and the server responded with a status code
+          // that falls out of the range of 2xx
+          if (!!hash && error.response.status === 404) {
```

Comment:
Open to suggestions here. Maybe we should make sure the hash is in the correct format, so that edge cases don't get caught here. Or maybe this is sufficient.

Reply:
I believe it's ok